app/client/src/tests/functions/UserActions.js

```diff
@@ -59,8 +59,8 @@ class UserActions {
     await this.page.waitForSelector("#user");
 
     // Enter info into fields
-    await this.page.type("#user", this.idir);
-    await this.page.type("#password", this.password);
+    await this.page.type('input[name="user"]', this.idir);
+    await this.page.type('input[name="password"]', this.password);
```

Comment:
Why? They haven't changed the id's.

Reply:
i will switch it back

Reply:
i didn't understand it initially so i switched it